pkgs/applications/editors/vscode/extensions/default.nix

```diff
@@ -1861,6 +1876,22 @@ let
         };
       };
 
+      randomfractalsinc.vscode-data-preview = buildVscodeMarketplaceExtension {
+        mktplcRef = {
+          name = "Data Preview";
+          publisher = "Random Fractal Inc";
+          version = "2.3.0";
+          shar256 = "DaI1/6IMQ44HrBM9tc04DGZmdlLdk1lVVMkMCa6cMHA=";
+        };
+        meta = with lib; {
+          changelog = "https://github.com/RandomFractals/vscode-data-preview/blob/master/CHANGELOG.md";
+          description = "Data preview for mulitple file extensions";
```

Comment:
```suggestion
          description = "A VS Code extension for data preview supporting multiple file extensions";
```

Reply:
In case you want to provide a long description, there is a `longDescription` field.